spec/graphql-crystal/language/parser_spec.cr

```diff
@@ -147,21 +146,22 @@ describe GraphQL::Language::Parser do
     end
   end
 
-  it "parses empty arguments" do
-    strings = [
-      "{ field { inner } }",
-      "{ field() { inner }}",
-    ]
+  # TODO: should it behave this way?
```

Comment:
seems only the first version is valid, empty `()` are not .

Reply:
Seems like empty () is not valid according to spec: https://graphql.github.io/graphql-spec/June2018/#sec-Appendix-Notation-Conventions
>A subscript suffix “Symbol list” is shorthand for a list of one or more of that symbol.
https://github.com/shopgun/graphql-erlang/issues/77#issuecomment-340203227

Reply:
ok, then the test should be reactivated for the `"{ field { inner } }"` case..

Reply:
done